### ISSUE TITLE
Only child pseudo selector doubt !!!

### ISSUE THREAD

**Issue by PSriVarshan** [flukeout/restaurant#274]
Why ``` plate:only-child ``` doesnt work wthout space?

**Comment by tynu4ok** [comment]
for lvl 16 plate need type:
plate :only-child - here we have a space between "plate" and ":"

**Comment by PSriVarshan** [comment]
yeah but what does the space do ? Is CSS getting strictly typed ?

**Comment by marralesfios** [comment]
@PSriVarshan

`plate:only-child` means the *plate* that is an only child

`plate :only-child` is the same as `plate *:only-child` which means any *descendant* of a plate that is an only child of its parent element
